docs/docs/user-setup-cli.mdx

```diff
@@ -0,0 +1,278 @@
+---
+title: (EXPERIMENTAL) Installing ZMK
+sidebar_label: (EXPERIMENTAL) Installing ZMK
+---
+
+This is an **experimental** command line program to help set up ZMK. [Click here to return to the main documentation](user-setup.mdx).
+
+ZMK CLI walks you through installing ZMK and setting up a GitHub repository to store and build custom firmware. It also automates some common tasks such as adding new keyboards to your repository.
+
+The instructions below contain commands that need to be run in a terminal program. On Windows, use the [Windows Terminal](https://apps.microsoft.com/detail/9n0dx20hk701) or PowerShell. On other operating systems, the terminal program is usually just named "Terminal".
+
+:::info
+If you use this program, any bug reports are highly appreciated. We are happy to help you fix your setup if you encounter a bug through the usage of this program.
+:::
+
+## Installation
+
+### Preliminaries
+
+#### Make sure you have Git installed
+
+```sh
+git --version
```

Comment:
Would it be useful to include a short explanation that running this command will print a version number if Git is installed or an error message if it is not? This may not be self-evident to people who aren't familiar with the command line.